twisted/conch/test/test_cftp.py

```diff
@@ -555,9 +555,8 @@ def makeFile(self, path=None, content=b''):
         """
         if path is None:
             path = self.mktemp()
-        file = open(path, 'w')
-        file.write(content)
-        file.close()
+        with open(path, 'w') as file:
```

Comment:
It might make more sense to rewrite things like this as `FilePath(path).setContent(content)`. (Note: I haven't looked at the surrounding context of any of the instances of this to see if there might be reasons that isn't appropriate in a particular case.)